Gems/Atom/Feature/Common/Assets/Shaders/Materials/DepthPass_WithPS.azsli

```diff
@@ -92,16 +91,25 @@ struct PSDepthOutput
 
 PSDepthOutput MainPS(VSDepthOutput IN, bool isFrontFace : SV_IsFrontFace)
 {
-    const float3 vertexNormal = normalize(IN.m_normal);
+    float3 vertexNormal = IN.m_normal;
+    float3 vertexTangent = IN.m_tangent;
+    float3 vertexBitangent = cross(vertexNormal, vertexTangent);
```

Comment:
We shouldn't do the cross product here because ConstructTBN will set it. We don't have to initialize any of these three vectors, actually.